addons/block_code/ui/script_window/script_window.gd

```diff
@@ -0,0 +1,14 @@
+@tool
+extends Window
+class_name ScriptWindow
```

Comment:
@GustavoLR548 please remove the class name. You are already preloading this script fine in `addons/block_code/block_code_plugin.gd` using the resource path. The problem we found with `class_name` was that they can collide with the user project. Suppose a user has a `ScriptWindow` class in their project and then they add this plugin, the names collide.
```suggestion
```